static/styles/_components/_filters.scss

```diff
@@ -59,12 +68,42 @@
   }
 }
 
+#filters {
```

Comment:
Can we stay away from id selectors? They increase specificity which in turn requires more specificity to override them, leading to a specificity war that could destroy the whole earth as we know it.